tests/components/grid.test.ts

```diff
@@ -1405,6 +1424,13 @@ describe("Copy paste keyboard shortcut", () => {
     return event;
   }
 
+  function getClipboardVisibleZones(model: Model): Zone[] {
```

Comment:
That's a duplicate of an existing helper function, right ?
You can move it to the `getters_helpers.ts`